packages/core/package.json

```diff
@@ -75,6 +76,7 @@
     "koa-send": "^5.0.1",
     "ky": "^1.2.3",
     "lru-cache": "^10.0.0",
+    "mime-types": "^2.1.35",
```

Comment:
this one looks better https://github.com/broofa/mime

Reply:
OK, will check it out